utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,3 @@
+const { PREDICTION_URL, CURRENT_USER } = require('sharify').data
+
+export const liveAuctionUrl = (id) => `${PREDICTION_URL}/${id}${CURRENT_USER ? '/login' : ''}`
```

Comment:
Regarding your comment, I would suggest using `CurrentUser.orNull()`, but be sure to guard against conditions where it is returned `null`. Also, I would avoid putting logic inside of template tags; something like this is more readable, even if its at the expense of a few more lines of code:

```javascript
export function liveAuctionUrl (id) {
  const currentUser = CurrentUser.orNull()

  if (currentUser) {
    return `${PREDICTION_URL}/${id}/login`
  } else {
    return '/some-fallback-url-so-you-dont-have-to-guard-everywhere-this-function-is-used'
  }
}